src/Mod/PartDesign/Gui/TaskPocketParameters.cpp

```diff
@@ -291,6 +353,193 @@ void TaskPocketParameters::onOffsetChanged(double len)
     recomputeFeature();
 }
 
+void TaskPocketParameters::fillDirectionCombo()
+{
+    bool oldVal_blockUpdate = blockUpdate;
+    blockUpdate = true;
+
+    if (axesInList.empty()) {
+        ui->directionCB->clear();
+        // add sketch normal
+        PartDesign::ProfileBased* pcFeat = static_cast<PartDesign::ProfileBased*>(vp->getObject());
+        Part::Part2DObject* pcSketch = dynamic_cast<Part::Part2DObject*>(pcFeat->Profile.getValue());
+        if (pcSketch)
+            addAxisToCombo(pcSketch, "N_Axis", QObject::tr("Sketch normal"));

```

Comment:
Why qualify `tr` here? You don't on the next line.

Reply:
Thanks. Fixed now